tutorials/space_shooter/README.md

```diff
@@ -1,3 +1,55 @@
-# Flame tutorial
+# Flame Tutorial
```

Comment:
This file might better be merged with doc/tutorials/tutorials.md, so that the people could see both "in-house" tutorials and the list of external ones.

Reply:
Agree